mifosng-provider/src/integrationTest/java/org/mifosplatform/integrationtests/LoanDisbursementDetailsIntegrationTest.java

```diff
@@ -0,0 +1,563 @@
+package org.mifosplatform.integrationtests;
```

Comment:
after adding a new file , please ensure that the gradle license task is run (this adds the license header to each new file, without which the tests would fail on Cloudbees)